### DISCUSSION TITLE
Keycloak Django integration - Authentication and Authorization

### DISCUSSION THREAD

**Discussion by RicardoFernandez-UY** [keycloak/keycloak#13407] [Q&A]
Hi,
Could someone please provide instruction on how to integrate Keycloak to a Django x project? 
I'm using Keycloak 13.x and Django 4.x

Are there are any adapters?, or maybe with the generic OIDC adapter?
I'm new to Keycloak and new to Django, == in trouble :-)

Any help/advice will be much appreciated!!
Thank you!!

**Comment by abstractj** [comment]
@RicardoFernandez-UY answered here https://github.com/keycloak/keycloak/discussions/13510. Please do not create multiple posts about the same topic.
